docs/networking/rocm.rst

```diff
@@ -0,0 +1,119 @@
+the current version.
+
+Since UCX will be providing the ROCm support, it is important to
+ensure that UCX itself is built with ROCm support.
+
+To see if your ucx was built with ROCm support run the following
+command:
+
+.. code-block:: sh
+
+ # Check if ucx was built with ROCm support
+ shell$ ucx_info -v
+
+ # configured with: --with-rocm=/opt/rocm --without-knem --without-cuda
+
+If you need to build ucx yourself to include ROCm support, please see
+the UCX documentation for `building ucx with Open MPI:
+<https://openucx.readthedocs.io/en/master/running.html#openmpi-with-ucx>`_
+
+It should look something like:
+
+.. code-block:: sh
+
+  # Configure UCX with ROCm support
+  shell$ ./configure --prefix=/path/to/ucx-rocm-install \
+                     --with-rocm=/opt/rocm --without-knem
+
+  # Configure Open MPI with UCX and ROCm support
+  shell$ ./configure --with-rocm=/opt/rocm    \
+         --with-ucx=/path/to/ucx-rocm-install \
+	  <other configure params>
+
+/////////////////////////////////////////////////////////////////////////
+
+How do I verify that Open MPI has been built with ROCm support?
+---------------------------------------------------------------
+
+Verify that Open MPI has been built with rocm using ``ompi_info``
+
+.. code-block:: sh
+
+   # Use ompi_info to verify ROCm support in Open MPI
+   shell$ ./ompi_info |grep "MPI extensions"
+          MPI extensions: affinity, cuda, ftmpi, rocm
+
+/////////////////////////////////////////////////////////////////////////
+
+
+How do I use ROCm-aware UCX for Open MPI?
+-----------------------------------------
+
+If UCX and Open MPI have been configured with ROCm support, specifying
+the ucx pml component is sufficient to take advantage of the ROCm
+support in the libraries. For example, running the ``osu_latency``
+from the `OSU benchmarks
+<https://mvapich.cse.ohio-state.edu/benchmarks>`_ with ROCm buffers
+using Open MPI and UCX ROCm support,
+
+.. code-block::
+
+   shell$ mpirun -n 2 --mca pml ucx \
+           ./osu_latency -d rocm D D
+
+Note: some additional configure flags are required to compile the OSU
+benchmark with ROCm buffers. Please refer to `UCX ROCm instructions
+<https://github.com/openucx/ucx/wiki/Build-and-run-ROCM-UCX-OpenMPI>`_
+for details.
+
+
+/////////////////////////////////////////////////////////////////////////
+
+Can I tell at compile time or runtime whether I have ROCm-aware support?
+------------------------------------------------------------------------
+
+Open MPI starting from v5.0 includes a function
+``MPIX_Query_rocm_support`` to check the availability of ROCm support
+in the library. To access the function, you need to include
+``mpi-ext.h``. Note that ``mpi-ext.h`` is specific to Open MPI.
+
+/////////////////////////////////////////////////////////////////////////
+
+What collective component should I use for ROCm support in collective operations
+----------------------------------------------------------
```

Comment:
I know this is kinda stupid, but Sphinx requires that the underlying be at least as long as the text title above it.